pkg/httpserver/crud/create_test.go

```diff
@@ -0,0 +1,77 @@
+package crud_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	configMocks "github.com/justtrackio/gosoline/pkg/cfg/mocks"
+	"github.com/justtrackio/gosoline/pkg/httpserver"
+	"github.com/justtrackio/gosoline/pkg/httpserver/crud"
+	logMocks "github.com/justtrackio/gosoline/pkg/log/mocks"
+	"github.com/justtrackio/gosoline/pkg/mdl"
+	"github.com/justtrackio/gosoline/pkg/validation"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestCreateHandler_Handle(t *testing.T) {
+	model := &Model{
+		Name: mdl.Box("foobar"),
+	}
+
+	config := configMocks.NewConfig(t)
+	config.EXPECT().GetDuration("crud.release_delay", 10*time.Minute).Return(10 * time.Minute)
+
+	logger := logMocks.NewLoggerMock(logMocks.WithMockAll, logMocks.WithTestingT(t))
+	transformer := NewTransformer()
+
+	transformer.Repo.On("Create", mock.AnythingOfType("*exec.stoppableContext"), model).Run(func(args mock.Arguments) {
```

Comment:
When moving / modifying tests, you could think about replacing `.On("Foo")` with `.EXPECT().Foo()`